clientSide/womeneducation/src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,45 @@
+.line
+{
+    margin-left: 100px;
+}
+.heartsymbolsize
+{
+    width:15px;
+    height:13px;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+.footertext
+{
+    font-weight: bold;
+    text-align: center;
+}
+.line
+{
+    font-size:17px;
+}
+.heartsymbolsize
+{
+    width:15px;
+    height:13px;
```

Comment:
line 5-11 is same as line 21-27 

please remove one

Reply:
I removed one class